Source/utils/display.h

```diff
@@ -89,8 +89,8 @@ void LogicalToOutput(T *x, T *y)
 
 	float scaleX;
 	SDL_RenderGetScale(renderer, &scaleX, NULL);
-	*x = static_cast<T>(*x * scaleX);
-	*y = static_cast<T>(*x * scaleX);

```

Comment:
this should've been `*y` on the right hand side

Reply:
lol!

Reply:
Fixed :)

https://github.com/diasurgical/devilutionX/commit/46c8e550ba30ac1391d6d015a7e37a4411755f5a